<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginf</class>
 <widget class="QDialog" name="loginf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Next Destination</string>
   </property>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>158</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_id"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>90</y>
      <width>222</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mot de passe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_mdp"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_statut">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>connecting...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
